rabbit-escape-render/test/rabbitescape/render/TestVector2D.java

```diff
@@ -0,0 +1,125 @@
+package rabbitescape.render;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.*;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.junit.Test;
+
+import rabbitescape.engine.CellularDirection;
+
+public class TestVector2D
+{
+    @Test
+    public void Vectors_add()
+    {
+        Vector2D a = new Vector2D( 1, 2 );
+        Vector2D b = new Vector2D( 3, 4 );
+        assertThat( a.add( b ), equalToVector2D( new Vector2D( 4, 6 ) ) );
```

Comment:
I don't think you need `equalToVector2D`.  I think if you use `equalTo` it will just work.